frontend/src/components/playlists/PlaylistsList.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import { Playlist } from './types';
+import { PlaylistItem } from './PlaylistItem';
+import { Box } from '@material-ui/core';
+
+interface PlaylistsListProps {
+  playlists: Playlist[];
+  onPlaylistSelected?: (playlist: Playlist) => void;
+}
+
+export const PlaylistsList: FC<PlaylistsListProps> = ({ playlists, onPlaylistSelected }) => {
+  return (
+    <>
+      {playlists.map((playlist, idx) => {
+        return (
+          <Box
+            key={idx}
+            mt={2}
```

Comment:
`mt` ? `mb` ?

Reply:
This is specific to  `material-ui` https://material-ui.com/system/basics/#material-ui-system, `m` stands for `margin` and `t` for `top` `mb` === `margin bottom` and so on...